docs/introduction/Car-registry-tutorial.rst

```diff
@@ -309,29 +305,28 @@ AbstractRegularTransaction
 
 Basic functionality is implemented for building required unlockers for input Regular boxes and returning a list of output Regular boxes according to input parameter *outputRegularBoxesData*. Also, basic transaction semantic validity is checked here. 
 
-
 CarDeclarationTransaction 
 *************************
 
 *CarDeclarationTransaction* extends previously declared *AbstractRegularTransaction* in the following way: ``public final class CarDeclarationTransaction extends AbstractRegularTransaction``
-newBoxes() -- a new box with a newly created car shall be added as well. Thus that function shall be overridden as well for adding new CarBox additional to regular boxes.  
+newBoxes() -- a new box with a newly-created car shall be added as well. Thus, that function shall be overridden as well for adding new CarBox additional to regular boxes.  
```

Comment:
@zennermac 

newBoxes() -- a new box for a new car must be added as well. This function will be overridden by adding a new CarBox to the RegularBoxes.